src/styles/_supported-device-table.scss

```diff
@@ -173,6 +173,7 @@ $row-padding: 8px;
         0 1px 1px 0 rgb(16 27 37 / 10%),
         0 1px 8px 0 rgb(46 69 82 / 12%);
       position: absolute;
+      bottom: -13px; //  (64px row - 30px button - 8px padding) / 2
```

Comment:
Can we use sass or css math to compute this instead of the comment?

Reply:
Yep, good call, thanks.

Updated in https://github.com/seamapi/react/pull/581/commits/5e67dc045b65ee18d3d4105aa67bad7973e3e515